### DISCUSSION TITLE
Is it possible to force skip package if repository unavailable.

### DISCUSSION THREAD

**Discussion by vol4onok** [composer/composer#11512] [Show and tell]
Hi Guys!
In composer.lock I have the package repository that was removed.
During the composer install I have an issue that `remote: Repository not found.`

Is it possible to ignore this issue and try to install other packages? Yes, I know it's a violation of the rules.

maybe somehow I can run composer install --force. something like this.

**Comment by stof** [comment]
there is no such feature in composer (and I don't think it would be accepted by maintainers)
